Applications/LLMEval/ContentView.swift

```diff
@@ -35,6 +35,7 @@ struct ContentView: View {
                     Spacer()
                     if llm.running {
                         ProgressView()
+                            .frame(maxHeight: 20)
```

Comment:
Set a height on this so that the enclosing view doesn't change size as it is running / not running (makes the UI jump around).